### PR TITLE
Bring back the source code to use serverStartPolicy to shutdown a ser…

### PR DESCRIPTION
**Pull request by hzhao-github**
…ver jenkins-ignore

### CODE REVIEW

**Comment by vanajamukkara** [comment]
fyi - I have this PR ready to be merged - https://github.com/oracle/weblogic-kubernetes-operator/pull/1123

You can use TestUtils.kubectlpatch() once its merged.

File: integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java [hunk 1]
```diff
@@ -2033,4 +2021,61 @@ public boolean accept(File dir, String name) {
     callShellScriptToBuildDeployAppInPod(
         appName, scriptName, username, password, infoDirName, archiveExt);
   }
+
```
Comment (vanajamukkara) [inline integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java]: If this code is not needed, should be removed.

Comment (hzhao-github) [reply integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java]: This trans doesn't touch that method and doesn't need to use that method either. It's used by TestUtils.java and then BaseTest and then ITMonitoringExporter.java

Comment (vanajamukkara) [reply integration-tests/src/test/java/oracle/kubernetes/operator/utils/Domain.java]: Its showing as you have commented these methods. See the changes in this PR under Files Changed.

**Comment by hzhao-github** [comment]
local run results:

06-25-2019 13:06:40 INFO   oracle.kubernetes.operator.ITSessionMigration staticUnPrepare SUCCESS
[INFO] Tests run: 2, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 412.503 s - in oracle.kubernetes.operator.ITSessionMigration
[INFO]

**Comment by hzhao-github** [comment]
Jenkins job passed

http://wls-jenkins.us.oracle.com/job/weblogic-kubernetes-operator-javatest/1831/consoleFull

**Comment by vanajamukkara** [comment]
You may want to sync to the latest develop branch as my PR got merged. We both touched the same code.

**Pull request merged**